File: src/BooKing.Web/src/app/dtos/reservationevents.dto.ts
export class ReservationEventsDto {
    id: string;
    eventType: string;
    createdAt: Date;
    eventTypeDescription: string;
    icon: string;
    additionalInformation: string;

    constructor(
      id: string,
      eventType: string,
      createdAt: Date,
      eventTypeDescription: string,
      icon: string,
      additionalInformation: string
    ) {
      this.id = id;
      this.eventType = eventType;
      this.createdAt = createdAt;
      this.eventTypeDescription = eventTypeDescription;
      this.icon = icon;
      this.additionalInformation = additionalInformation;
    }
}

File: src/BooKing.Reserve/Booking.Reserve.Application/Dtos/ReservationEventsDto.cs
namespace BooKing.Reserve.Application.Dtos;
public class ReservationEventsDto
{
    public Guid Id { get; set; }
    public string EventType { get; set; }
    public string EventTypeDescription { get; set; }
    public string Icon { get; set; }
    public DateTime CreatedAt { get; set; }
    public string AdditionalInformation { get; set; }
}
